javalib/src/main/scala/java/net/ProtocolFamily.scala

```diff
@@ -0,0 +1,5 @@
+package java.net
+
+trait ProtocolFamily {
```

Comment:
I do not see this in the JDK `java.net` documentation, so probably
needs to be private, probably private[net] 
(I believe that one can have private traits).

Reply:
It is part of the java public [API](https://docs.oracle.com/javase/8/docs/api/java/net/ProtocolFamily.html)

Reply:
Today I learned... Thank you for the URL, got any kitchen sinks hidden in there?